flink-table-store-core/src/main/java/org/apache/flink/table/store/CoreOptions.java

```diff
@@ -429,6 +438,10 @@ public int maxFileNum() {
         return options.get(COMPACTION_MAX_FILE_NUM);
     }
 
+    public Integer maxSortedRunNum() {
+        return options.get(COMPACTION_MAX_SORTED_RUN_NUM);
+    }
+
     public boolean enableChangelogFile() {
```

Comment:
Maybe we should set the default value of `numLevels` to `numSortedRunStopTrigger`.